Starting with 17.15.0, we're no longer able to use `frida -f /usr/bin/df`
because of broken imports. It seems some files were introduced without updating
the build system. Let's add `repl_inspect.py` to fix the obvious breakage, but
also other files that are in the source tree but not in the Meson definition
file.
--- a/subprojects/frida-tools/frida_tools/meson.build
+++ b/subprojects/frida-tools/frida_tools/meson.build
@@ -7,6 +7,7 @@ sources = [
   'compiler.py',
   'creator.py',
   'discoverer.py',
+  'itracer.py',
   'join.py',
   'kill.py',
   'ls.py',
@@ -18,8 +19,12 @@ sources = [
   'push.py',
   'reactor.py',
   'repl.py',
+  'repl_inspect.py',
   'rm.py',
+  'stracer.py',
+  'stream_controller.py',
   'tracer.py',
+  'units.py',
 ]
 
 python.install_sources(sources, subdir: 'frida_tools')
